### PR TITLE
Use trame's new server.context and translator instead of our implementation

### PR DESCRIPTION
**Pull request by alesgenova**

### CODE REVIEW

File: nrtk_explorer/app/core.py [hunk 1]
```diff
@@ -41,73 +42,53 @@ def image_id_to_result(image_id):
...
 
-        self._server = server
+        super().__init__(server)
 
-        self.server._local_state = {
-            "image_objects": {},
-        }
+        self.context["image_objects"] = {}
+        self.context["images_manager"] = images_manager.ImagesManager()
+        self.context["annotations"] = {}
```
Comment (jourdain) [inline nrtk_explorer/app/core.py:54]: you can use the `.` notation too.

Comment (alesgenova) [reply nrtk_explorer/app/core.py:54]: Yes, it's a leftover from the fact we were using a dict for the `local_state`.
So is the `.` notation preferred over `[]` when using a trame state object?

Comment (jourdain) [reply nrtk_explorer/app/core.py:54]: yes, it is a more natural way unless you want to do something like

```
for item in items:
   self.context[item] = ...
```

**Review by vicentebolea — COMMENTED** [review]
Looks good!!

**Pull request merged**